script/SubmitSafeTxs.s.sol

```diff
@@ -48,9 +49,10 @@ contract SubmitSafeTxs is Script {
 
     function run() external {
         // read transactions
+        string memory blockchainName = determineBlockchain().name;
         string memory path = string.concat(
-            vm.projectRoot(),
-            "/flat_output.json"
+            blockchainName,
+            "_flat_output.json"
```

Comment:
Note for us, to remove this hard-coding  `_flat_output.json` introduced by me :)